static/app/views/dashboards/widgets/common/types.tsx

```diff
@@ -19,13 +21,4 @@ export interface StateProps {
   onRetry?: () => void;
 }
 
-export type MaxValues = {
-  max1: number;
-  max2: number;
-};
-
-// `max_values` is Snake Case to preserve compatibility with the current widget serializer. We _do_ want to change it to Camel Case!
-export interface Thresholds {
-  max_values: MaxValues;
-  unit: string;
-}
+export type Thresholds = ThresholdsConfig;
```

Comment:
nit: Should we just drop this type and use only `ThresholdsConfig` instead? Looks like we only reference `Thresholds` in one area.